backend/src/utils/adminUtils.ts

```diff
@@ -65,114 +72,40 @@ export const getAllowedUserList = async (fastify: KubeFastifyInstance): Promise<
   );
 };
 
-export const getGroupsConfig = async (
-  fastify: KubeFastifyInstance,
-  customObjectApi: CustomObjectsApi,
-  username: string,
-): Promise<boolean> => {
-  try {
-    const auth = getAuth();
-    const adminGroupsList: string[] = auth.spec.adminGroups;
-
-    if (adminGroupsList.includes(SYSTEM_AUTHENTICATED)) {
-      throw new Error('It is not allowed to set system:authenticated as admin group.');
-    } else {
-      return await checkUserInGroups(fastify, customObjectApi, adminGroupsList, username);
-    }
-  } catch (e) {
-    fastify.log.error(e, 'Error getting groups config');
-    return false;
-  }
+const SingletonAuthResource: V1ResourceAttributes = {
+  group: 'services.platform.opendatahub.io',
+  resource: 'auths',
+  name: 'default-auth',
```

Comment:
Is this guaranteed to always be the name?

Reply:
Yup -- currently what we doing -- the patch from the client assumes this too (RHOAI 2.17 release)